InnerEye/ML/Histopathology/models/deepmil.py

```diff
@@ -115,6 +119,11 @@ def get_pooling(self) -> Tuple[Callable, int]:
         num_features = self.num_encoding*self.pool_out_dim
         return pooling_layer, num_features
 
+    def get_dropout(self) -> Callable:
```

Comment:
Maybe we could also add a function for hidden layers? We could add an optional hidden layer before the dropout and classifier in `forward()`

Reply:
As we discussed, I'm moving dropout into `get_classifier()` and we can incorporate the hidden layer option in a future PR.